Docs/RAG/FVA_Pipeline.md

```
@@ -0,0 +1,321 @@
+  "falsification_triggered_count": 1,
+  "status_changes": {
+    "unverified->refuted": 1
+  },
+  "total_time_ms": 2650.8,
+  "budget_exhausted": false
+}
+```
+
+### Get FVA Settings
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)